docs/welcome.mdx

```diff
@@ -18,11 +18,12 @@ Flashbots is a research and development organization formed to mitigate the nega
 ## Get started
 
 <Grid>
-<GridBlock symbol="I." title="Searchers">
+<GridBlock symbol="I." title="Searchers & Telegram Bots">
```

Comment:
I think searchers should remain their own category here. We could create an entirely separate box on this page for "applications" in the future -- but I think that should be its own PR. I'd revert this change.